docs/packages/web/README.md

```diff
@@ -254,12 +252,52 @@ module.exports = {
       // Example: change the page title
       html: {
         title: 'Epic Web App'
+      },
+
+      // Example: Proxy webpack-dev-server requests to http://localhost:3000
+      devServer: {
+        proxy: 'http://localhost:3000'
       }
     }]
   ]
 };
 ```
 
+### Dev Server Proxy
+
+If your are integrating Neutrino into an existing app, you may want to set up a proxy for development. See Webpack's [`devServer.proxy`](https://webpack.js.org/configuration/dev-server/#devserver-proxy) for all available options.
```

Comment:
Webpack should be lower-case.

Reply:
Also, let's change "existing app" to something like "developing against a server". I'm not for sure.